services/github/src/handlers.js

```diff
@@ -523,9 +548,9 @@ async function statusHandler(message) {
         output: {
           title: `${this.context.cfg.app.statusContext} (${eventType.split('.')[0]})`,
           summary: `${taskDefinition.metadata.description}`,
-          text: `[Task group](${taskGroupUI(this.context.cfg.taskcluster.rootUrl, taskGroupId)})`,
+          text: `[${CHECKRUN_TEXT}](${taskUI(this.context.cfg.taskcluster.rootUrl, taskGroupId, taskId)})\n${customCheckRunText || ''}`,
         },
-        details_url: taskUI(this.context.cfg.taskcluster.rootUrl, taskGroupId, taskId),
+        details_url: taskGroupUI(this.context.cfg.taskcluster.rootUrl, taskGroupId),
```

Comment:
why send details to the group instead of the task now? it feels like that should still be the task as well?

Reply:
I want to switch the two URLs. Right now, the most useful URL (the one that leads straight to the task) appears in small gray font on the bottom of the checkrun :) And the less useful URL (to the taskgroup) that requires additional clicking to get to the task appears as a proper larger link. I am noticing that many users don't even realize that that small gray thing is _there_. So I decided to switch them

Reply:
Nevermind, switched them back. It doesn't matter, and it's out of scope of this PR anyways